NEWS.md

```diff
@@ -24,6 +24,10 @@ v1.4.0, TBD
 - Add support for multi-iov RMA read and aromic operations
 - Allow multi-recv to post buffer larger than message size limit
 
+## RxD provider notes
+
+- Initial release
```

Comment:
Ok, I will remove this commit then. Do you want the man page also to be removed for this release?